src/components/UI/Text/InputNumber.vue

```diff
@@ -152,7 +152,7 @@ export default {
     localValue: {
       // local copy of the value prop
       get() {
-        return this.value;
+        return this.value.toString();
```

Comment:
@dalmia are we sure we should do this?

Reason - 
this is an InputNumber component. The value that the user types in the textbox gets saved as a `Number` data type which makes sense because this is a InputNumber component.

We need to convert it into string only when doing this check `this.localValue.length == this.maxLength`

If we go ahead with converting the `localValue` into string as what Saloni did in her latest commit, then we're forever converting this number into a string and that is what gets emitted into the parent.

Currently it should work fine because we're not using this component anywhere else other than Login page but this doesn't feel correct given this is an inputNumber component. The component should hold an internal data as Number not a string.

Reply:
@Salonichoudhary07 I talked to Aman regarding this. He agrees with what I said here. 
The first commit you made, where you did `this.localValue.toString().lenght == this.`, we should do that change.

Also one more point of feedback that Aman asked me to give you, is that, if someone has reviewed your PR and asked something, that doesn't imply that it is the right answer. You should be confident of the change you made and if you're not, play around with things enough so that you're confident. In this case for example, Aman asked a question "If doing that would be better?". So you should defend the changes you've made :p

Reply:
Yes, I will do that.